### PR TITLE
Ragini/internal public page scrollbar design/test

### PR DESCRIPTION
**Pull request by ViaSocket-Git**

### CODE REVIEW

File: src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss [hunk 1]
```diff
@@ -121,13 +118,13 @@
   width: 320px;
   border-right: 1px solid grey;
   height: 100vh;
-  overflow: scroll;
+  overflow-y: scroll;
```
Comment (Saurabh186) [inline src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss:121]: use auto

File: src/components/publishSidebar/publishSidebar.scss [hunk 2]
```diff
@@ -52,7 +52,8 @@
 
 .publish-sidebar-container {
   padding: 30px;
-  overflow: scroll;
+  overflow-y: scroll;
```
Comment (Saurabh186) [inline src/components/publishSidebar/publishSidebar.scss:55]: use auto

**Review by Saurabh186 — CHANGES_REQUESTED** [review]

File: src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss (hunk 1 above)
Comment (ViaSocket-Git) [reply src/app/(other)/orgs/[orgId]/collection/[collectionId]/redirections/redirections.scss:121]: solve

File: src/components/publishSidebar/publishSidebar.scss (hunk 2 above)
Comment (ViaSocket-Git) [reply src/components/publishSidebar/publishSidebar.scss:55]: solve

**Review by Saurabh186 — APPROVED** [review]

**Review by Saurabh186 — APPROVED** [review]

**Pull request merged**